apps/frontend/src/pages/project/libs/components/setup-analytics-modal/setup-analytics-modal.tsx

```diff
@@ -54,10 +65,19 @@ const SetupAnalyticsModal = ({
 					<Input
 						control={control}
 						errors={errors}
-						isDisabled
+						isDisabled={!hasProjectApiKey}
+						isReadOnly={hasProjectApiKey}
```

Comment:
let's use readonly for both cases
```suggestion
						isReadOnly={true}
```